Sources/StreamChat/Database/DTOs/MessageDTO.swift

```diff
@@ -1309,72 +1309,43 @@ private extension ChatMessage {
 
         if let currentUser = context.currentUser {
             isSentByCurrentUser = currentUser.user.id == dto.user.id
-            $_currentUserReactions = ({
```

Comment:
what's happening here, can you share more details? :)

Reply:
Diff looks pretty unreadable here.
Old approach captured a closure which was fed into CoreDataLazy. CoreDataLazy then ran this closure. Since it is a property wrapper then the call site looks a bit weird. But this line triggers CoreDataLazy's projectedValue.

Now we run the code immediately here instead.